unit_tests/testLinearSolver/testLinearSolver.f90

```diff
@@ -2794,7 +2799,7 @@ SUBROUTINE testIterativeSolve_GMRES()
 
   !set iterations and convergence information and build/set M
   SELECTTYPE(thisLS); TYPE IS(LinearSolverType_Iterative)
-    CALL thisLS%setConv(2_SIK,1.0E-9_SRK,1.0E-9_SRK,1000_SIK,30_SIK)
+    CALL thisLS%setConv(2_SIK,1.0E-6_SRK,1.0E-6_SRK,100_SIK,3_SIK)
```

Comment:
Don't need _SIK's, even though they were already there.

Reply:
SIK's and SNK's have been purged from the linear solver test files